### PR TITLE
UI: Users

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2020-02-05 at 4 44 37 PM
Screen Shot 2020-02-05 at 4 44 35 PM

### CODE REVIEW

**Comment by cesnietor** [comment]
When you hover over description or just type somethin a gray component on the left starts flickering.
Screen Shot 2020-02-06 at 11 20 10 AM

**Comment by cesnietor** [comment]
Description should be email
Screen Shot 2020-02-06 at 11 20 46 AM

File: portal-ui/src/screens/Console/Users/Users.tsx [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+                {records.map(row => (
+                  <TableRow key={row.name}>
+                    <TableCell className={classes.wrapCell}>
+                      {row.name}
+                    </TableCell>
+                    <TableCell className={classes.wrapCell}>
+                      {row.email}
+                    </TableCell>
+                    <TableCell align="right">
+                      {/*<IconButton*/}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Users/Users.tsx]: please delete this commented code

File: portal-ui/src/screens/Console/Users/AddUser.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+            )}
+          </Grid>
+          {addError !== "" && (
+            <Grid item xs={12}>
+              <Typography
+                component="p"
+                variant="body1"
+                className={classes.errorBlock}
+              >
+                {`${addError}`}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Users/AddUser.tsx]: please change `{`${addError}`}` to {addError}

File: portal-ui/src/screens/Console/Users/DeleteUser.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                        Are you sure you want to delete user{" "}<b>{selectedUser.name}</b>?
+                        {deleteError !== "" && (
+                            <React.Fragment>
+                                <br />
+                                <Typography
+                                    component="p"
+                                    variant="body1"
+                                    className={classes.errorBlock}
+                                >
+                                    {`${deleteError}`}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Users/DeleteUser.tsx]: please change `{`${deleteError}`}` to `{deleteError}`

**Review by Alevsk — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Users/Users.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Users/Users.tsx]: done

File: portal-ui/src/screens/Console/Users/AddUser.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Users/AddUser.tsx]: done

File: portal-ui/src/screens/Console/Users/DeleteUser.tsx (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Users/DeleteUser.tsx]: done

**Review by Alevsk — DISMISSED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**